tests/testthat/helper-scope.R

```diff
@@ -58,12 +58,6 @@ renv_tests_scope_repos <- function(scope = parent.frame()) {
 
 }
 
-renv_tests_scope_system_cache <- function(scope = parent.frame()) {
```

Comment:
The point of this was to use the system cache for installing bioconductor packages to speed the tests up.

Reply:
Yeah, but the Bioconductor tests are failing now and I don't understand why. 😞